_posts/2017-07-05-jenkins-job-builder.md

```diff
@@ -0,0 +1,125 @@
+---
+title: Introduction to Jenkins Job Builder
```

Comment:
I might title this with something more specific like "Testing Jenkins Configurations with Jenkins Job Builder"